libcloudforensics/aws.py

```diff
@@ -818,11 +934,28 @@ def CreateVolumeCopy(zone,
         'Account')
 
     if source_account_id != destination_account_id:
+      if volume_to_copy.encrypted:
+        # Generate one-time use KMS key that will be shared with the
+        # destination account.
+        kms_key_id = source_account.CreateKMSKey()
```

Comment:
These two `CreateKMSKey` and `ShareKMSKeyWithAWSAccount` can also raise `RuntimeError`, so please udpate the docstring accordingly. It might be a good idea to raise more specific errors. Opened #71 to track this.